openapi/src/resources/azure_vm.yaml

```diff
@@ -17,18 +17,10 @@ components:
         vmImage:
```

Comment:
Don't you also want to delete region at line 11?

Reply:
Azure resources still have a region. I have only removed the parameters to creation APIs that allow the user to specify region since region comes from the landing zone.

This probably applies to all your other comments about region.